tests/smoke.rs

```diff
@@ -0,0 +1,31 @@
+#[mixed_tag_serde::mixed_tags(Serialize, Deserialize)]
+#[derive(serde::Deserialize, serde::Serialize, Debug, PartialEq, Eq)]
+#[serde(untagged)]
+enum MyEnum {
+    #[serde(rename = "one")]
+    #[tagged]
+    One(i32),
+
+    #[serde(rename = "two")]
+    #[tagged]
+    Two(u32),
+
+    #[tagged]
```

Comment:
Shouldn't it be `#[untagged]`/no annotation at all instead?

Reply:
Of course, I just tested it for throwing compile error on misusing the tag